### PR TITLE
Branch 3

### PR DESCRIPTION
**Pull request by Paskab012**

### CODE REVIEW

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @KABIKA681, 

Good job so far!

## Highlight!
- Good commit messages ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the review comments below ._

- [ ] Good job setting up the linters.  However, stylelint is failing. Please try fixing the error by clicking on the `details` beside the failing linter to be able to see the errors and then go to your index.css file and fix the errors 😄.

- [ ] Consider making your Pull Request description more descriptive by adding a descriptive title telling you what changes are inside. and also a short summary describing introduced changes in more details. Check out this [link ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) to learn how to write a good pull request description.

- [ ] Consider adding `node_modules` directory to `.gitignore` as it can be added by accident while setting up linters.

- [ ] I suggest you stop closing pull requests before a review as it makes the files changed to not include all the files you changed. This makes it hard for code reviewers to see all the files changed and all your commit messages 😄

- [ ] Please consider giving your feature branch a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). This is to make your branch name more descriptive and make your repository more professional.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Hi @KABIKA681 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.

Highlights:
      
 ## Required Changes ♻️

As mentioned by the reviewer before, please, make corrections in the following:

- [x] Consider making your Pull Request description more by adding a descriptive title

- [x] Consider adding node_modules directory to .gitignore

- [ ] Consider giving your feature branch a name of a feature or action 

- [x] Pull Request should have a short summary describing introduced changes in more detail


## Optional suggestions

- n/a

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md [hunk 1]
```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
 
 [Live Demo Link](http://127.0.0.1:5500/)
 
-
 ## Getting Started
 
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```
Comment (Elerqsousy) [inline README.md]: these instructions are for you to understand how to use the readme file. next time delete any instructions and it should look better.

File: README.md [hunk 2]
```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
...
 
-
 ### Prerequisites
```
Comment (Elerqsousy) [inline README.md:15]: this section is to explain to the visitor how to use your code, and not to explain how you did it.

File: README.md [hunk 3]
```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
...
+
 - You have a <Windows/Linux/Mac> machine. State which OS is supported/which is not.
 - Good internet connection.
+
 ### Setup
+
 - Create a Github repository and name it "Hello-microverse""
 - Add a small description, a Readmefile and lisence(MIT)
 - After creating the Repo, clone the repo into your Pc and open the file into your editor.
 - Configurate Linters
```
Comment (Elerqsousy) [inline README.md]: once linters are configured once on a repo, all other users don't need to install it again.

File: README.md [hunk 4]
```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
...
+
 ### Install
+
 - The latest version of <a given code editor(Vscode, Atome etc)>
+
 ### Usage
 
-👤 ***Author**
+👤 **\*Author**
```
Comment (Elerqsousy) [inline README.md]:
```suggestion
👤 ***Pascal Kabika**
```

File: README.md [hunk 5]
```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
 
 [Live Demo Link](http://127.0.0.1:5500/)
```
Comment (Elerqsousy) [inline README.md]: this should be a live demo link of your project. You can do that by clicking on the settings of your branch and then on the left side menu click on pages and configure your live page. If you don't want a live demo, just consider deleting this section.

**Review by Elerqsousy — COMMENTED** [review]
great job my friend.

**Comment by Paskab012** [comment]
## Hello @elmar8287 
I apologize for the inconvenience; I would like to ask permission to redo my entire project due to the instructions that I did not apprehend well when reading the question yesterday. Instead of creating a single feature branch, I have created three, thinking that each action would be on its feature branch.
That's why I beg you to give me the chance to start over to do a good project in the end.
Best regard.

**Comment by julie-ify** [comment]
Hi @KABIKA681 

Please ignore the changes that require you to rename your feature branch to a more descriptive feature branch name as this should be implemented starting from your next projects. Please make other requested changes on this same pull request and push and submit again for review.  Please do not open a new pull request 😄. Sorry for any inconveniences 🙏

**Comment by Paskab012** [comment]
## Last project setup 
in this last setup I:
- Have added the node_modules into the .gitignore file
- Have a little bit corrected the readme file for better user undestanding.

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @KABIKA681 , 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**